h/notification/domain_mailer.py

```diff
@@ -1,109 +0,0 @@
-# -*- coding: utf-8 -*-
-import re
-import logging
-from urlparse import urlparse
-
-import requests
-from bs4 import BeautifulSoup
-from pyramid.events import subscriber
-from pyramid.renderers import render
-from pyramid.security import Everyone, principals_allowed_by_permission
-
-from h import events
-from h.notification.gateway import user_profile_url, standalone_url
-from h.notification.notifier import send_email, TemplateRenderException
-from h.notification.types import ROOT_PATH
-
-log = logging.getLogger(__name__)
-
-TXT_TEMPLATE = ROOT_PATH + 'document_owner_notification.txt'
-HTML_TEMPLATE = ROOT_PATH + 'document_owner_notification.html'
-SUBJECT_TEMPLATE = ROOT_PATH + 'document_owner_notification_subject.txt'
```

Comment:
As long as it doesn’t break notification system we use in https://github.com/webplatform/annotation-service/blob/master/notes_server/archiver.py